bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/models/form/Review.java

```diff
@@ -0,0 +1,30 @@
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ ~ Copyright 2023 Adobe
+ ~
+ ~ Licensed under the Apache License, Version 2.0 (the "License");
+ ~ you may not use this file except in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~     http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing, software
+ ~ distributed under the License is distributed on an "AS IS" BASIS,
+ ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~ See the License for the specific language governing permissions and
+ ~ limitations under the License.
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+package com.adobe.cq.forms.core.components.models.form;
+
+import org.osgi.annotation.versioning.ConsumerType;
+
+/**
+ * Defines the {@code Review} Sling Model used for the {@code /apps/core/fd/components/form/review} component.
+ *
+ * @since com.adobe.cq.forms.core.components.models 2.1.0
+ */
+@ConsumerType
+public interface Review extends Base {
+    String[] getLinkedPanels();
```

Comment:
The interface should be of provider type

please add java doc to all the methods. What does String[] array mean, if  it is qualified name, then please link the QN documentation in the java doc